### DISCUSSION TITLE
Would this work with react native on iOS/Android?

### DISCUSSION THREAD

**Discussion by marcoschwartz** [xyflow/xyflow#1860] [Q&A]
Hello,

Just wondering - would this project work in mobile app made using react native? Thanks!

**Comment by raarts** [comment]
Looked into this: no, will not work, regular HTML elements like div are not supported on React Native. 
I ended up using a WebView which connects to a webserver. This works pretty well, especially with v10, because of better touch support.

**Reply by fbartho** [reply]
@raarts do you still use this solution?

> which connects to a webserver.

I assume this means you have a live service for this info, rather than a webview that is directly powered from the app?
